pyaedt/edb_core/padstack.py

```diff
@@ -462,14 +462,21 @@ def set_solderball(self, padstackInst, sballLayer_name, isTopPlaced=True, ballDi
         return False
 
     @pyaedt_function_handler()
-    def create_coax_port(self, padstackinstance, use_dot_separator=True):
+    def create_coax_port(self, padstackinstance, use_dot_separator=True, name=None):
         """Create HFSS 3Dlayout coaxial lumped port on a pastack
         Requires to have solder ball defined before calling this method.
 
         Parameters
         ----------
         padstackinstance : `Edb.Cell.Primitive.PadstackInstance` or int
             Padstack instance object.
+        use_dot_separator : bool, optional
+            Whether to use ``.`` as the separator for the naming convention, which
+            is ``[component][net][pin]``. The default is ``True``. If ``False``, ``_`` is 
+            used as the separator instead.
+        user_defined_port_name : str
+            if a name is provided will overwrite the default naming convention. If a port is already defined with the
+            same name, default naming will be reverted.
```

Comment:
```suggestion
            Port name for overwriting the default port-naming convention,
            which is ``[component][net][pin]``. The port name must be unique.
            If a port with the specified name already exists, the
            default naming convention is used so that port creation does
            not fail.
```
This description should match the earlier one for ``port name``. It seems that the parameter name should also be consistent--but oh well!